src/index.html

```diff
@@ -21,7 +21,436 @@
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- end header -->
+
+    <!-- product -->
+    <main class="container">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <h3 class="card__subtitle">Product code: 195434</h3>
+
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+
+          <p class="stars__title">Reviews: 5</p>
+        </div>
+
+        <p class="card__price">
+          <span class="card__price-label">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </p>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <h3 class="card__subtitle">Product code: 195434</h3>
+
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+
+          <p class="stars__title">Reviews: 5</p>
+        </div>
+
+        <p class="card__price">
+          <span class="card__price-label">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </p>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
```

Comment:
The 'data-qa' attribute is used for test selectors and should not be present in the production code unless it is being used for automated testing.